src/Infrastructure/LeanCode.AzureIdentity/AzureCredentialConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+namespace LeanCode.AzureIdentity;
+
+public record AzureCredentialConfiguration
+{
+    public string? ClientId { get; init; }
+    public string? TenantId { get; init; }
```

Comment:
I'd reorder this to match the order of initializers in `DefaultLeanCodeCredential.Create`, so `TenantId` before `ClientId`.

Same with keys below.

Reply:
Only half done.